libparsec/crates/platform_mountpoint/src/windows/mod.rs

```diff
@@ -0,0 +1,495 @@
+// Parsec Cloud (https://parsec.cloud) Copyright (c) BUSL-1.1 2016-present Scille SAS
+
+mod error;
+
+use anyhow::anyhow;
+use once_cell::sync::{Lazy, OnceCell};
+use std::{
+    ops::Deref,
+    path::{Path, PathBuf},
+    sync::{Arc, Mutex},
+};
+
+use winfsp_wrs::{
+    filetime_from_utc, filetime_now, u16cstr, CleanupFlags, CreateFileInfo, CreateOptions,
+    FileAccessRights, FileAttributes, FileInfo, FileSystem, FileSystemContext, PSecurityDescriptor,
+    Params, SecurityDescriptor, U16CStr, U16CString, VolumeInfo, VolumeParams,
+    WriteMode as WinWriteMode, NTSTATUS, STATUS_DIRECTORY_NOT_EMPTY, STATUS_NOT_IMPLEMENTED,
+    STATUS_RESOURCEMANAGER_READ_ONLY,
+};
+
+use libparsec_types::FileDescriptor;
```

Comment:
```suggestion
use libparsec_types::prelude::*;
```

and then Bam ! You also got `anyhow` exposed for free too ^^

Reply:
Done